### ISSUE TITLE
 [RTL] The back icon on the URL bar is reversed

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6542]
See screenshot:
https://drive.google.com/file/d/1mLtWMCfvF_3lrZXtJcWrHfQxIaB4DvfP/view

The arrow should be facing right.

via : https://bugzilla.mozilla.org/show_bug.cgi?id=1634932

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2397)

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on main 228b77b9a89f133417cb748fcf720fb01e6b6f1a

**Comment by SimonBasca** [comment]
Verified fix on v98.0 (8023).
